app/src/components/preloaders/progressLoader/progressLoader.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
preloaders folder for for different preloaders components, not for progressbars. Moreover, that progress bar not looks like reusable and styled for Import modals needs. So I think it will be better to move that component directly into folder with modal import